tests/mock_clients/mock_azureblob.py

```diff
@@ -106,8 +106,15 @@ def content_as_bytes(self):
 
 
 class MockContainerClient:
-    def __init__(self, root):
+    def __init__(self, root, container_name):
         self.root = root
+        self.container_name = container_name
+
+    def exists(self):
+        if self.container_name == "container":  # name used by passing tests
```

Comment:
Wondering if this would be more clear if we had this as a global constant, and had conftest instead import it from here. Right now these are just hardcoded values in two different places.

Reply:
Good call, did this for all the defaults.